### PR TITLE
CB: Travis enablement

### PR DESCRIPTION
**Pull request by drostron**

### CODE REVIEW

File: it/src/test/scala/quasar/fs/ReadFilesSpec.scala [hunk 1]
```diff
@@ -201,28 +201,9 @@ object ReadFilesSpec {
...
-      Data.Obj(ListMap("seq" -> Data.Int(i), "filler" -> jsonTree(3)))
-
-    TestDatum(
-      readsPrefix </> file("large"),
-      EStream.range(1, numDocs) map (json))
-  }
-
-  val veryLongFile = TestDatum(
+  val largeFile = TestDatum(
     readsPrefix </> dir("length") </> file("very.long"),
```
Comment (mossprescott) [inline it/src/test/scala/quasar/fs/ReadFilesSpec.scala]: Probably should change the name to "large" at this point, just to avoid confusion.

**Comment by mossprescott** [comment]
LGTM but needs merge from master.

**Comment by mossprescott** [comment]
The test times actually look pretty decent in Travis, although overall it's pretty slow, so I guess it’s the inserts?

**Comment by drostron** [comment]
Might be. It would be good to collect timings of the various operations at some point to confirm or deny assumptions. Thankful to finally have CB integration tests running in Travis at least.

**Comment by drostron** [comment]
Ready for another look.

**Pull request merged**